### DISCUSSION TITLE
Tabby being flagged as Malicious on VirusTotal

### DISCUSSION THREAD

**Discussion by stukinnear** [Eugeny/tabby#9699] [General]
Hi

Not sure if you're aware of this - but our Security guy has just mentioned to me that a company called GridinSoft has flagged Tabby as malicious on VirusTotal.

Hope you can put them right?

Stu

**Comment by Eugeny** [comment]
Which release file is that?

**Reply by stukinnear** [reply]
1.0.197

https://www.virustotal.com/gui/file/a9223bb9d9326964654baa179cde859f11cdd743f4a578a9121e7a191bc45d4e/details
